src/commandHandlers/help.ts

```diff
@@ -19,15 +19,9 @@ export const command = async (arg: [string, string], embed: MessageEmbed) => {
       false
     );
   } else {
-    let text = '';
-    commandList.forEach((commandItem) => {
-      text = text
-        .concat(`**${COMMAND_PREFIX}${commandItem.triggers[0]}**  `)
-        .concat(`${commandItem.description}\n`)
-        .concat(
-          'Usage:' + '`' + `${COMMAND_PREFIX}${commandItem.usage}\n` + '`'
-        );
-    });
+    const format = commandItem =>
+      '`' + `${COMMAND_PREFIX}${commandItem.triggers[0]}` + '`';
+    const text = commandList.map(format).join(' | ');
```

Comment:
```suggestion
    const format = (commandItem: any) =>
      '`' + `${COMMAND_PREFIX}${commandItem.triggers[0]}` + '`';
    const text = commandList.map(format).join(' | ');
```

This will fix the issue for now.

@eddiejaoude we should look at building an interface for commands, though.

Reply:
Yes we really should 👍 . I usually have TS on super strict, but I didn't want to make it too difficult for people

Interfaces would definitely help 🤓

Reply:
It can come in another PR though

Reply:
#334 discusses this(though not sure if command interface should be made now, or after the API)